### DISCUSSION TITLE
not show imodel by sample app

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#2874] [Q&A] [answered]
now,i run the   https://github.com/iTwin/mobile-sdk-samples，and open the local imodel file
 can show the imodel view list but not show the model 

![1_20211209151326](https://user-images.githubusercontent.com/18022720/145351132-0fd8c9ba-007a-4d3d-93e0-135e44157a60.jpg)

![2_20211209150237](https://user-images.githubusercontent.com/18022720/145351179-2bfc6d8d-3707-4851-8aa8-c3015b18827f.jpg)

**Comment by Carl-Hinkle** [comment, marked as answer]
Hi, can you please share the .bim file?  We've tested multiple files on several devices, including an iPad Pro (11 inch) (3rd Generation) and we're not seeing any issues.

Also, are there any errors being logged in either Xcode or in the Safari developer tools?

**Reply by paprophet** [reply]
> Hi, can you please share the .bim file? We've tested multiple files on several devices, including an iPad Pro (11 inch) (3rd Generation) and we're not seeing any issues.
> 
> Also, are there any errors being logged in either Xcode or in the Safari developer tools?

give me the email address ,i will send the bim file to you . my email 173988573@qq.com

**Reply by Carl-Hinkle** [reply]
Sent you an email.

**Reply by Carl-Hinkle** [reply]
Hi, have you tried opening the "Views" drawer and selecting one of the items from the list?  Or using the "Fit View" tool under the vertical meatball menu in the upper right of the viewport?  I still haven't received the .bim file from you and we haven't been able to replicate the problem as you describe.

**Reply by tcobbs-bentley** [reply]
Note that many iModels have a default view that does not show anything. If your file is that way, it would explain the behavior you see. The "Fit View" command will show geometry if there is any geometry enabled in the default view. Otherwise, hopefully one of the other views can be selected, and then "Fit View" will work on it. We are modifying the behavior of the sample app in the future to generate the default view instead of relying on the (unreliable) one built into the iModel.

**Reply by Carl-Hinkle** [reply]
I'm going to mark this as answered since it's been nearly a month since there was an update.  If this is still causing issues please open a new discussion.
